openpype/hosts/aftereffects/api/extension/jsx/hostscript.jsx

```diff
@@ -252,7 +252,7 @@ function _getItem(item, comps, folders, footages){
     return JSON.stringify(item);
 }
 
-function importFile(path, item_name, import_options){
+function importFile(path, item_name, import_options, fps){
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function